pep/translations/en-us.json

```diff
@@ -335,7 +335,7 @@
             },
             "relatedDocuments": {
                 "title": "Related Documents",
-                "noResults": "You must view a document to see the documents related to it"
+                "noResults": "You must view a document to see the documents related to it. If you are already viewing one, this means no results have been found"
```

Comment:
i think it would be cleaner to have this message be conditional. show 
> You must view a document to see the documents related to it.
if there is no related document currently set in the sidebar data, otherwise show
> No related documents found.

But we dont need to hold up the PR for this if you dont want to do it now, think it should be a polish item eventually tho.

Reply:
FWIW - I had the same conversation in my head